packages/gatsby/src/commands/build-html.js

```diff
@@ -1,6 +1,8 @@
 /* @flow */
 const webpack = require(`webpack`)
 const fs = require(`fs`)
+const { promisify } = require(`util`)
+const promisifiedUnlink = promisify(fs.unlink)
```

Comment:
honestly I would move to fs-extra and use remove insetad of the promise dance

Reply:
Good point since we already have fs-extra. Done!